src/ASM/LR/ASMu2Dmx.C

```diff
@@ -908,3 +908,17 @@ void ASMu2Dmx::getBoundaryNodes (int lIndex, IntVec& nodes, int basis,
     for (size_t b = 1; b <= this->getNoBasis(); ++b)
       this->ASMu2D::getBoundaryNodes(lIndex, nodes, b, thick, orient, local);
 }
+
+
+void ASMu2Dmx::remapErrors(Vector& errors, const Vector& origErr) const
+{
+  const LR::LRSplineSurface* basis = this->getBasis(1);
+  const LR::LRSplineSurface* geo = this->getBasis(ASMmxBase::geoBasis);
+
+  for (const LR::Element* elm : basis->getAllElements()) {
+    int gEl = geo->getElementContaining((elm->umin()+elm->umax())/2.0,
+                                        (elm->vmin()+elm->vmax())/2.0) + 1;
+    for (const LR::Basisfunction* b: elm->support())
```

Comment:
Cosmetics: space between "b:" for readability

Reply:
done